src/main/java/com/codejoust/main/Main.java

```diff
@@ -25,7 +25,8 @@ public static void main(String[] args) {
 		SpringApplication.run(Main.class, args);
 
 		// Initialize Firebase library for user authentication actions
-		String keyfile = System.getenv(FIREBASE_KEYFILE);
+		// String keyfile = System.getenv(FIREBASE_KEYFILE);
+		String keyfile = "{\"type\":\"service_account\",\"project_id\":\"rocketden-tester\",\"private_key_id\":\"1a09d0deef1be40de7388899a6d464ed3c6cfb1f\",\"private_key\":\"-----BEGIN PRIVATE KEY-----\\nMIIEvQIBADANBgkqhkiG9w0BAQEFAASCBKcwggSjAgEAAoIBAQC7MmVHuGnmsf8X\\nXibCOqaqD7VKT3ldPLp2ogpSD+gFeBxRsFmXwHTyg0bqo0IDMvn8+KRkCOZnKiN7\\nPS94Z+Duk0CUpWdfadgq2DKLT5BTzDvVw1CX5bf5j3AHnSOIRObazCxaVfXEvk65\\na72RXmWbYmZoq/hRdaWstQFgr7mfqtydk23z9elv/zBEoZ0wihdigrZIpXn9VMWV\\nq6MF/IJmYjkdSJkLSgmGLOGW4h9aMTNUKcvhYNkmy7Aac85tccEazi1Gds/4w+Y4\\npCRCiBM7MFOdzAMJib1x0Kek2ZdTILcDHvSrfxzi/urL+IbhHxlD5wdbweLaviuE\\nZKP1Mnb3AgMBAAECggEAU6iHg8BiL5O/rUlJ6AZ/IkRH4FBZLDZuHIN5OkhaiEsv\\n2MAHoh6EBTdu84CHiVd4SjoyeXRSx6sY1UQ4WUaPSv+q3mN3KbhrP2aEXCbwGMsO\\nuI8Djwa+K/0S84wh643jX8fcUHjXD5h/JTWFsBgwL3uPydqE7J63rlHsLjIKMlTM\\nl4s0S5B/Q0SLf6HABDFL09LxuxIZBBr0Kclh8a4s6ocWF18Oiz0rH1vxDUrhBaZz\\ncOB8iHHWeYm09z2l0brpsAV8bYAv+djqxcv2jUfQTKk9i5E+BMdCK5sgCto0k5Eg\\no/xUUKpr+Oq/W5TohMRn9ZJmKo+U+xD4Sf6nUy4lMQKBgQDw7rhJaOEbroC155xQ\\n5RkHPLaFrDHnHNwdSdb8jCjCZjN//6C7wgowoAhVTX5lkF9Qf++2W6sDh0uNdLfM\\nj/LDoOL18TuWPCVZ4xxRIxZnQBnFNdunK+JfNA33noGsdNnlf6kCU9CnCOn3b1Mh\\nmr93Lw7+42RRrSp33AiVBK88hwKBgQDG52EM9oyVHlxO+2hS312JTO1TPK1vm0s5\\nwpSbDSQF62xp19I2qBDgVke3efmXATfd2A9vWJRESyx3MixrhwTB6wwbis5ZGiz1\\nT8S3tuYEhO6Q6o+OO8m4IY+F5AjGqYVRBVeoYRsvElfmxuG7NzNTK9MBUPpgx6Tm\\ndn/JilN+EQKBgQCoSCTh6WZp3k26QGa3Pf1oR6i43xgP8+Fud5e41/DUL2xDN/o9\\nin9xjSWk5zakm91nLaTx+z8MfOBxkheawuRubZ546CCVg7cW/IFXzqkkPuqBC2g7\\ncHRxyLsEvGqYWRmkqbETlhVDxf8dO12WOC8JgQ35TDxSrB/gjghUhWwcWwKBgDfh\\nUVa+/Yf3W5+k6g4+Sp0ES7iS13b/fgbgL7fP3Ur8dRzXUWuhk/wywTs+vWw7H2Lo\\nDI4PqLf3Fqr2IZyj9iY0ztX+DTLjM7ZHjtC+RakHX66+903SZj2g0B2hjwWhIp88\\njP/1KBn1YVSok0pxm63GJiW8hlsiO6WzrPsGLabBAoGAIyzfgIZDspXhSveNJpb4\\nnmPR2tzDzpJvS+xd0DLmEce+XUVvrLVQdntxrIDwo6StbqLs6thg7C/JIOzbdVZb\\nMUD+2xvT88PAdi1zhU6q4+Ng4wkaxcI7IgB2KnBd/63D5w2mOHC80kLc26A60+WO\\nWRBlXKvY6XSPzntScn1TTxM=\\n-----END PRIVATE KEY-----\\n\",\"client_email\":\"firebase-adminsdk-7ltnz@rocketden-tester.iam.gserviceaccount.com\",\"client_id\":\"111327020402649497499\",\"auth_uri\":\"https://accounts.google.com/o/oauth2/auth\",\"token_uri\":\"https://oauth2.googleapis.com/token\",\"auth_provider_x509_cert_url\":\"https://www.googleapis.com/oauth2/v1/certs\",\"client_x509_cert_url\":\"https://www.googleapis.com/robot/v1/metadata/x509/firebase-adminsdk-7ltnz%40rocketden-tester.iam.gserviceaccount.com\"}";
```

Comment:
nit: undo this before merging - also, since this is in the commit history, I went ahead and revoked this keyfile + generated a new one (located with the rest of our secrets), which you and @zpChris will want to update locally in order to allow the service to boot up properly (edit: the new keyfile will prob have to wait for now based on what we talked about during standup)